tsdb/chunkenc/float_histogram.go

```diff
@@ -211,18 +220,15 @@ func (a *FloatHistogramAppender) AppendHistogram(int64, *histogram.Histogram) {
 // Appendable returns whether the chunk can be appended to, and if so
 // whether any recoding needs to happen using the provided interjections
 // (in case of any new buckets, positive or negative range, respectively).
+// If the sample is a gauge type, AppendableGauge must be used instead.
 //
 // The chunk is not appendable in the following cases:
 //
 // • The schema has changed.
-//
 // • The threshold for the zero bucket has changed.
-//
 // • Any buckets have disappeared.
-//
 // • There was a counter reset in the count of observations or in any bucket,
 // including the zero bucket.
-//
 // • The last sample in the chunk was stale while the current sample is not stale.
```

Comment:
I think we needed the empty lines because godoc would otherwise munge everything into one paragraph.

By now, godoc supports proper lists, though, see https://pkg.go.dev/github.com/fluhus/godoc-tricks#Lists

So you _can_ remove the empty lines, but you have to replace the `•` with a `-` and indent the lines.